backend/src/test/java/gov/cdc/usds/simplereport/service/PersonServiceTest.java

```diff
@@ -542,22 +555,99 @@ void deletePatient_adminUser_success() {
                 PATIENT_PAGESIZE,
                 ArchivedStatus.UNARCHIVED,
                 null,
-                false)
+                false,
+                "")
             .size());
 
     List<Person> result =
         _service.getPatients(
-            null, PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, ArchivedStatus.ALL, null, false);
+            null, PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, ArchivedStatus.ALL, null, false, "");
     assertEquals(1, result.size());
     assertTrue(result.get(0).isDeleted());
     assertEquals(
         1,
         _service
             .getPatients(
-                facilityId, PATIENT_PAGEOFFSET, PATIENT_PAGESIZE, ArchivedStatus.ALL, null, false)
+                facilityId,
+                PATIENT_PAGEOFFSET,
+                PATIENT_PAGESIZE,
+                ArchivedStatus.ALL,
+                null,
+                false,
+                "")
             .size());
   }
 
+  @Test
+  @WithSimpleReportSiteAdminUser
+  void deletePatient_siteAdminUser_success() {
+    makedata(false, false);
+    UUID site1Id = _site1.getInternalId();
+    Person charles =
+        _service
+            .getPatients(
+                site1Id, 0, 5, ArchivedStatus.UNARCHIVED, null, false, _org.getExternalId())
+            .get(0);
+    _service.setIsDeleted(charles.getInternalId(), true, _org.getExternalId());
+
+    List<Person> result =
+        _service.getPatients(
+            site1Id,
+            PATIENT_PAGEOFFSET,
+            PATIENT_PAGESIZE,
+            ArchivedStatus.UNARCHIVED,
+            null,
+            false,
+            _org.getExternalId());
+    assertEquals(3, result.size());
+    assertPatientList(result, BRAD, ELIZABETH, AMOS);
+  }
+
+  @Test
+  @WithSimpleReportSiteAdminUser
+  void deletePatient_siteAdminUser_noExternalOrgId_throwsException() {
+    makedata(false, false);
+    UUID site1Id = _site1.getInternalId();
+    Person charles =
+        _service
+            .getPatients(
+                site1Id, 0, 5, ArchivedStatus.UNARCHIVED, null, false, _org.getExternalId())
+            .get(0);
+    UUID charlesId = charles.getInternalId();
+    MisconfiguredUserException caught =
```

Comment:
⚠️ this exception is what gets thrown when we try and get the current org for a user and they don't have an org -- which a site admin (unless assigned to or ghosted into an org) won't have